reconcile/test/utils/test_gql.py

```
@@ -92,3 +102,100 @@ def test_gqlapi_throws_gqlapierrorforbiddenschema_exception(
+    )
+    result = gql_api.query.__wrapped__(gql_api, SIMPLE_QUERY)  # type: ignore[attr-defined]
+    assert result["__typename"] == "Query"
+
+
+def test_gqlapi_query_full_stack_with_variables(httpserver: HTTPServer) -> None:
+    httpserver.expect_request("/graphql", method="POST").respond_with_json({
+        "data": {"user": {"name": "test-user"}}
+    })
+    gql_api = GqlApi(
+        httpserver.url_for("/graphql"),
+        token="Basic test-token",
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to argument: "token"

(S106)